### DISCUSSION TITLE
Build app with internal packages.

### DISCUSSION THREAD

**Discussion by phamduytien1** [vercel/turborepo#6751] [Help] [answered]
Hi, how can I build my typescript node app with internal packages? I build but always give me `"SyntaxError: Cannot use import statement outside a module"` .
In the example of turbo, I see it is using "transpilepackages" of nextjs, is there any alternative ways?
Thanks in advance.

**Comment by gtjamesa** [comment]
I've just spent a couple of days stuck on the same problem and I think I've finally found a solution that works well, I've posted some details here: https://github.com/vercel/turbo/discussions/4509#discussioncomment-7822830

**Comment by anthonyshew** [comment, marked as answer]
We've added some documentation on Internal Package strategies that should help in answering this question: https://turbo.build/repo/docs/core-concepts/internal-packages
